components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { adminMenu } from "./admin-menu";
+
+import { LicenseContext } from "../license-context";
+import { ReactElement, useContext, useEffect } from "react";
+import { getGitpodService } from "../service/service";
+
+import { ReactComponent as Alert } from "../images/exclamation.svg";
+import { ReactComponent as Success } from "../images/tick.svg";
+import { ReactComponent as Tick } from "../images/check.svg";
+import { ReactComponent as Cross } from "../images/x.svg";
+
+export default function License() {
+    const { license, setLicense } = useContext(LicenseContext);
+
+    useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
+        }
+        (async () => {
+            const data = await getGitpodService().server.adminGetLicense();
+            setLicense(data);
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, msg, tick] = getSubscriptionLevel(
+        license?.plan || "",
+        license?.userCount || 0,
+        license?.seats || 0,
+        license?.fallbackAllowed || false,
+    );
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod Installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card className="bg-gray-800 dark:bg-white text-white dark:text-gray-400">
+                        <p className="text-white dark:text-black font-bold pt-4 text-sm"> {licenseLevel}</p>
+                        <p className="dark:text-gray-500">{paid}</p>
+                        <p className="text-gray-400 pt-4 pb-2">Available features:</p>
+                        {features &&
```

Comment:
**issue**: This is minor and could be ignored for this iteration, but these are fetched from ther backend but feels like there's a typo for some of them. Could we safely rename the following or create follow up issues for cleaning up these?

| BEFORE | AFTER |
|-|-|
| Prebuild | Prebuilds |
| Set Timeout | Timeouts |
| Snapshot | Snapshots |

Reply:
This is a great question! I think I will create another issue for this and deal with it outside the scope, since it might require other code changes in the server/licensor components (I am getting the feature names from an enum in licensor). I don't know if I want to hack more on the front end for this.

Reply:
Hooking into this conversation: It would be great if we can match the naming here to what is on the pricing page - e.g. it’s workspace sharing here in the admin UI but shared workspaces in the pricing page. Not urgent but makes things tidier.

Reply:
Opened https://github.com/gitpod-io/gitpod/issues/9785 to track this discussion and avoid discussions on closed PRs. 🏓 Cc @nandajavarma @lucasvaltl